cmd/storage-rest-common.go

```diff
@@ -16,8 +16,10 @@
 
 package cmd
 
-const storageRESTVersion = "v9"
-const storageRESTPath = minioReservedBucketPath + "/storage/" + storageRESTVersion + SlashSeparator
+const (
+	storageRESTVersion = "v10"
```

Comment:
It is probably better to keep the API and just register error type ?

Reply:
Error type cannot be registered for errrors.errorString there is no real error.

Reply:
This change is expected..

Reply:
Reverted pushing a new change..